pom.xml

```diff
@@ -2,18 +2,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.smartnews</groupId>
-    <artifactId>maven-jpa-entity-generator-plugin</artifactId>
```

Comment:
As I mentioned above, I don't see the necessity to have two ways (gradle and maven) to build the core library.